test/system/static.test.ts

```diff
@@ -13,17 +14,20 @@ const getFixture = (fixturePath) =>
 test("static analysis builds the expected response", async (t) => {
   const thisIsJustAnImageIdentifierInStaticAnalysis = "node:doesnotexist";
   const dockerfile = undefined;
+  const tmpDirPath = tmpdir();
```

Comment:
Does this automatically cleanup the directory? If not we should remove it after the test is done.

Reply:
The files created by the plugin are cleaned up once done with them: https://github.com/snyk/snyk-docker-plugin/blob/master/lib/inputs/rpm/static.ts#L33-L35
Note that `tmpdir()` does not create a directory but returns the path to the OS temporary directory (usually /tmp)

Reply:
As long as the test leaves nothing behind it is ok ...